doc/documentation-main.html

```diff
@@ -2000,6 +2000,13 @@ <h3 class="sourcetitle">testng.xml</h3>
 You might want to surround your expression with a <tt>CDATA</tt> declaration (as shown above) to avoid tedious quoting of reserved XML characters).<br>&nbsp;
 </li>
 </ul>
+<b>Note:</b>
+<p class="prettyprint">
+  Starting from version <b>7.5</b> TestNG does not bring in any dependency on BeanShell implementations by default. <br>
```

Comment:
Maybe you should add that now TestNG supports all scripts supporting the spi we use.

Reply:
I will raise a separate PR which adds that information because I think I will also need to add some working samples that can be referred to by people.